api/websocket.go

```diff
@@ -18,12 +18,10 @@ func (api *API) InitWebSocket() {
 }
 
 func connect(c *Context, w http.ResponseWriter, r *http.Request) {
-	originChecker := utils.GetOriginChecker(r)
```

Comment:
I don't know why `GetOriginChecker` took an argument. It doesn't use it in any way.